app/views/lots/_lot.html.slim

```diff
@@ -1,14 +1,16 @@
 .col-lg-4
   .card.border-0.rounded.shadow-sm.mb-3 class=(lot.published? ? 'pub-lot-card' : 'unp-lot-card')
     .lot-image-container
-      = image_tag(lot.images[0].url, class: 'card-img-top') if lot.images?
-      .lot-action-buttons-container
-        = link_to "edit", edit_lot_path(lot), class: 'lot-action-button ml-3 link px-3 font-weight-bold'
-        - if lot.published?
-          = link_to "unpublish",unpublish_lot_path(lot), method: :put, remote: true, class: 'js-lot-status-btn lot-action-button link px-3 font-weight-bold'
-        - else
-          = link_to "publish",publish_lot_path(lot), method: :put, remote: true, class: 'js-lot-status-btn lot-action-button link px-3 font-weight-bold'
-        = link_to "delete", lot_path(lot), method: :delete, class: 'lot-action-button mr-3 link px-3 font-weight-bold'
+      - if lot.images?
+        = link_to image_tag(lot.images[0].url, class: 'card-img-top'), lot_path(lot)
+      - if is_private
```

Comment:
IMHO, `is_private` isn't clear enough. What does it mean private lot? Like published/unpublished?
Something like `own_lot` would be better I think